### PR TITLE
challenge finished

### PR DESCRIPTION
**Pull request by Han-Zhang-fuze**
Animation

### CODE REVIEW

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+.backgroundimg {
+  background-image: linear-gradient(rgb(0 0 0 / 20%), rgb(0 0 0 / 30%)), url("images/hero.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  margin-left: auto;
+}
+
+.col-one-half {
+  width: 60%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:71]: This name doesn't match its measurement 🤔

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+  color: white;
+  line-height: 1.6;
+  font-family: Avenir, sans-serif;
+  font-size: 15px;
+}
+
+.col-one-half > button {
+  border-radius: 5px;
+  width: 200px;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:90]: I would just create a class so that this styling is reusable

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+#btn {
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:92]: Use `class`es, not `id`s

File: css-canvas-medical/styles.css [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+.locations {
+  background-color: #fafafa;
+  margin-top: 40px;
+  padding-top: 40px;
+  padding-bottom: 60px;
+}
+
+.loc-one-half {
+  width: 50%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:112]: Why not use a proper `column` class?

File: css-canvas-medical/styles.css [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+  font-family:
+    Avenir,
+    sans-serif;
+}
+
+.align-content-center {
+  text-align: center;
+  margin: auto;
+  width: 40%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:121]: Why not use a proper `column` class?

File: css-canvas-medical/styles.css [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+  font-weight: normal;
+  font-size: 15px;
+  padding: 10px;
+  color: #909090;
+}
+
+.container-width {
+  max-width: 80%;
+  margin: auto;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:149]: This should just be your plain `container` class

Comment (codingcodymiller) [reply css-canvas-medical/styles.css:149]: I see that this _already is_ your plain `container` class, on line 8. Be careful not to unnecessarily duplicate styling 👀

File: css-canvas-medical/styles.css [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+  border-radius: 5px;
+}
+
+.bot-one-half {
+  width: 50%;
+}
+
+.bottom {
+  display: flex;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:194]: Why not just use a `row`?

File: css-canvas-medical/styles.css [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+.footer {
+  background-color: #525351;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.left-content {
+  display: flex;
+  flex-direction: column;
+  width: 60%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:205]: Why not use a proper `column` class?

File: css-canvas-medical/styles.css [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+
+.left-content > img {
+  width: 150px;
+  margin-bottom: 40px;
+}
+
+.right-content {
+  display: flex;
+  color: white;
+  width: 40%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:218]: Why not use a proper `column` class?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-canvas-medical/index.html [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+    </div>
+  </div>
+  <div class="locations">
+    <div class="row justify-content-spacebetween">
+      <div class="align-content-center col-two-fifth">
+        <h2 class="adjust-heading-font-color-size-family-weight">Clinicians Prefer Canvas</h2>
+        <p class="adjust-subheading-color-line-height">Physician-owned primary care practices across the country have already made Canvas the fabric of their care.</p>
+      </div>
+    </div>
+    <div class="row container justify-content-spacebetween">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:61]: No one element should be both a `container` and a `row`. Instead, you should make an element with the `container` class that has a child with the `row` class

Comment (codingcodymiller) [reply css-canvas-medical/index.html:61]: This issue is present in multiple other places in your code, please make sure it is addressed in all instances.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Han-Zhang-fuze** [comment]
May-17-2022 15-21-17

**Comment by Han-Zhang-fuze** [comment]
May-17-2022 18-11-30

File: css-canvas-medical/index.html [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+          </li>
+          <li>
+            <a href="#">About Us</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li><button id="navbtn">Contact Us</button></li>
+        </ul>
+      </nav>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:36]: No `column` classes?

File: css-canvas-medical/index.html [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+  </div>
+  <div class="container">
+    <div class="row learn-more justify-content-spacebetween">
+      <div class="align-content-left">
+        <h3>Ready to learn more?</h3>
+        <p>Schedule a private meeting now</p>
+      </div>
+      <div class="align-content-right">
+        <button class="btn-right">Get Started</button>
+      </div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:84]: No `column` classes?

File: css-canvas-medical/index.html [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+    </div>
+  </div>
+  <div class="footer">
+    <div class="container row">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:88]: No one element should be both a `container` and a `row`, I believe I mentioned this in a previous review.

File: css-canvas-medical/styles.css [hunk 14]
```diff
@@ -0,0 +1,338 @@
...
+.left-content {
+  display: flex;
+  flex-direction: column;
+  color: white;
+  line-height: 1.6;
+}
+
+.col-three-fifth {
+  width: 60%;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:214]: Maybe a `column-two-thirds` with some padding 🤔

File: css-canvas-medical/styles.css [hunk 15]
```diff
@@ -0,0 +1,338 @@
...
+
+.product-img-left {
+  width: 100%;
+  padding-right: 80px;
+}
+
+.product-img-right {
+  width: 100%;
+  padding-left: 80px;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:265]: These seem like they should just be some padding utility classes added to a `column-full` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Fantastic work, Han!! This is all looking great, excellent revisions 👏 👏 👏

**Pull request merged**